.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,25 @@
+# Description of Changes
+<!-- Summarize your code change. -->
+
+
+# Motivation & Context
+<!-- Why this code change was made. -->
+
+
+# Type of change
+<!-- Mark all that apply (the fewer the better) -->
+- [ ] 🐛 Bug fix
+- [ ] ✨ New feature
+- [ ] 🛠 Technical currency
+- [ ] 💎 Code cleanliness
+
+# Snapshots
+<!-- If not applicable, remove this section. -->
+
+
+# Future considerations:
```

Comment:
I'd like to combine this and notes and just generalize them as Discussion. PRs would by and large be the way for us to talk about design choices and how they're gonna affect code changes, so an open ended area is really the only way to encapsulate everything.

Reply:
I see what you mean about setting a catch all "Discussion" section, but I do think we'd benefit from keeping a section that lays out next steps to be taken after each code change. Adding everything under "Discussion" feels too generic to me, but now that you mention it so does a "Notes" section...

What do you think about:
"Future considerations" => "Future Work", and 
"Notes" => "Discussion"

Reply:
Talking about future work in a PR that doesn't actually do that work just crowds and detracts the conversation from the code change itself. I don't think a PR is the place for documenting next steps. You can set up something else for that (a google doc, trello board, etc.) but I don't go back to closed PRs to look at planned work.

Reply:
Gotcha, updated in 41181cd71308b1628c9b0b048eb0a58b703c5f3b. I think a doc or trello board would be useful, we can scrap that other one I made in like a night. I need to headsdown on the mobile app base but can make a new board if you don't have the time.